testing/kuttl/e2e/pgbouncer/00-assert.yaml

```diff
@@ -5,9 +5,9 @@ metadata:
 status:
   instances:
     - name: instance1
-      readyReplicas: 2
-      replicas: 2
-      updatedReplicas: 2
+      readyReplicas: 1
+      replicas: 1
+      updatedReplicas: 1
```

Comment:
📝 #2983 changes the cluster to have `replicas: 2`. The tests don't use multiple replicas right now, so maybe this is best. 🌱 I suspect we'll include something with read-replicas eventually here.

Reply:
OK, made that change. (Also reverted the index change.)